optimism.tokenlist.json

```diff
@@ -179,6 +179,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+     {
+      "chainId": 42,
+      "address": "0x24a19ee5a5c8757acdebe542a9436d9c796d1c9e",
+      "name": "0xBitcoin",
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x4D772843C34E84dC8091cCe1484623445fBe6eC7",
+      "name": "0xBitcoin",
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 1,
+      "address": "0xb6ed7644c69416d67b522e20bc294a9a9b405b31",
+      "name": "0xBitcoin",
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x78eb06535651115e45853409aC24763932e24C06",
```

Comment:
Please update to newly deployed token at 0xe0BB0D3DE8c10976511e5030cA403dBf4c25165B

Reply:
I am confused. so the newly deployed test token address is 0xe0BB0D3DE8c10976511e5030cA403dBf4c25165B? or you want a new token deployed....